packages/admin-next/dashboard/src/routes/orders/order-create-claim/components/claim-create-form/schema.ts

```diff
@@ -0,0 +1,23 @@
+import { z } from "zod"
+
+export const ClaimCreateSchema = z.object({
+  inbound_items: z.array(
+    z.object({
+      item_id: z.string(),
+      quantity: z.number(),
+      reason_id: z.string().optional().nullable(),
+      note: z.string().optional().nullable(),
+    })
+  ),
+  outbound_items: z.array(
+    z.object({
+      item_id: z.string(), // TODO: variant id?
```

Comment:
comment: Yeah I think we need to be able to pass variants here. We don't have the item ID upfront, as far as I can see. We can tackle that when we add support for outbound items.